router/core/router.go

```diff
@@ -1709,7 +1743,7 @@ func WithClientHeader(cfg config.ClientHeader) Option {
 
 type ProxyFunc func(req *http.Request) (*url.URL, error)
 
-func newHTTPTransport(opts *SubgraphTransportOptions, proxy ProxyFunc) *http.Transport {
+func newHTTPTransport(opts TransportTimeoutOptions, proxy ProxyFunc) *http.Transport {
```

Comment:
This transport was created for the server. Not sure, if the same defaults should be applied to the subgraph client. What defaults did we use before?

Reply:
ok